atlp-rwanda/e-comm-team-brogrammers-fn
# #184948181 User makes a payment

#### What does this PR do?
This is a PR to show a user payment confirmation, of failure page
Some feature are not yet implement due to other requirements that are unavailable at the moment.
#### Description of Task to be completed?
Acceptance Criteria:

GIVEN that the buyer is on the payment page
WHEN the buyer enters their payment information and clicks the ""Pay"" button
THEN the payment should be processed via Stripe API and the buyer should be taken to a confirmation page
GIVEN that the buyer is on the confirmation page
WHEN the buyer views the confirmation
THEN the buyer should see a confirmation message with order details, including the order number, total cost, and expected delivery date, as well as a confirmation of the payment made via Stripe API

#### How should this be manually tested?
To test this head over to `/payment-success` or `/payment-failed`, and you will see the corresponding page.
#### Any background context you want to provide?
n/a
#### What are the relevant pivotal tracker/Trello stories?
https://www.pivotaltracker.com/n/projects/2639400/stories/184948181
#### Screenshots (if appropriate)
<img width="854" alt="Screenshot 2023-05-16 at 16 47 38" src="https://github.com/atlp-rwanda/e-comm-team-brogrammers-fn/assets/57767759/54723b6f-bea2-47f0-8291-8c11f06f6299">
<img width="643" alt="Screenshot 2023-05-16 at 16 48 05" src="https://github.com/atlp-rwanda/e-comm-team-brogrammers-fn/assets/57767759/4da3ef5f-7717-46c4-9bd9-9a48887e7b4a">

#### Questions:
n/a
